src/__tests__/actions/loginAction/__test__/loginAction.test.js

```diff
@@ -0,0 +1,85 @@
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import fetchMock from "fetch-mock";
+import loginAction from "../../../../actions/loginAction";
+import { LOGIN_SUCCESS, LOGIN_FAIL } from "../../../../actions/types";
+
```

Comment:
Remove unnecessary blank lines in your code

Reply:
I have removed them